playbooks/tasks/portal-health-check-disable.yml

```diff
@@ -12,7 +12,9 @@
   register: result
 
 - name: Stop portal healh check to take portal out of load balancer
-  command: 'docker exec health-check cli/disable "{{ out_of_lb_message }}"'
+  vars:
+    disable_message: "{{  out_of_lb_message | default('') }}"
+  command: 'docker exec health-check ./cli disable "{{ disable_message }}"'
```

Comment:
As Karol mentioned, we should force to use a message.
I would just keep an original command.

Reply:
Now I see the reason: disabling during deploys or restarts.
We can make it:

```- name: Stop portal healh check to take portal out of load balancer
  command: 'docker exec health-check ./cli disable "{{ out_of_lb_message | default(portal_action) }}"'
  when: result.exists and result.container is defined and result.container.State.Running```

`portal-action` var is defined for portal deploy (=`portal-deploy`), restart (=`portal-restart`), rollback (=`portal-rollback`) playbooks, so we will see the health check disable reason during these actions.